src/Time.cpp

```diff
@@ -5,22 +5,22 @@
 #include "../include/Time.h"
```

Comment:
Your changes are respected but you should focus on your class. If a function name is provided in the header file, use it regardless of its implementation.